station/app/api/api_v1/endpoints/local_trains.py

```diff
@@ -271,7 +227,7 @@ async def get_file(train_id: str, file_name: str):
     return Response(file)
 
 
-@router.get("/{train_id}/getLogs")
+@router.get("/{train_id}/logs", response_model=list[lt_schemas.LocalTrainLog])
```

Comment:
`response_model=List[lt_schemas.LocalTrainLog]`

use list type `List[]`instead of `list` which initializes a list

Reply:
Done